config/storage.yml

```diff
@@ -7,12 +7,12 @@ local:
   root: <%= Rails.root.join("storage") %>
 
 # Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
```

Comment:
We can remove this line since we're using `Figaro.env` now instead of `Rails.application.credentials`, right?